api/grpcserver/activation_service.go

```diff
@@ -0,0 +1,57 @@
+package grpcserver
+
+import (
+	context "context"
+	"fmt"
+
+	v1 "github.com/spacemeshos/api/release/go/spacemesh/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/spacemeshos/go-spacemesh/api"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+type activationService struct {
+	atxProvider api.AtxProvider
+}
+
+func NewActivationService(atxProvider api.AtxProvider) *activationService {
+	return &activationService{
+		atxProvider: atxProvider,
+	}
+}
+
+// RegisterService implements ServiceAPI.
+func (s *activationService) RegisterService(server *Server) {
+	log.Info("registering GRPC Activation Service")
+	v1.RegisterActivationServiceServer(server.GrpcServer, s)
+}
+
+// Get implements v1.ActivationServiceServer.
+func (s *activationService) Get(ctx context.Context, request *v1.GetRequest) (*v1.GetResponse, error) {
+	logger := log.GetLogger().WithFields(log.Binary("id", request.Id))
```

Comment:
Binary encodes with base64 i think, and everywhere in the codebase we represent ids in hex, so it would be very hard to match

Reply:
Fixed - I used `log.Stringer()` to serialize ATXID in hex.